doc/source/getting_started/licensing.rst

```diff
@@ -0,0 +1,83 @@
+.. _ref_licensing:
+
+=========
+Licensing
+=========
+
+This section details what the user should expect in terms of limitations or license usage
+when running PyDPF scripts.
+DPF follows a client-server architecture,
+which means that the PyDPF client library must interact with a DPF server.
+It thus either starts a DPF server via a local installation of DPF,
+or it connects to an already running local or remote DPF server.
+
+DPF is installed along with ANSYS since ANSYS 2021R1 (see :ref:`ref_compatibility`).
```

Comment:
The compatibility table should talk about 2024.2.pre0, right?

![image](https://github.com/ansys/pydpf-core/assets/107186344/285fba45-73ad-4e5b-ace5-40532f79b5a2)